source_test.yaml

```diff
@@ -256,15 +256,18 @@
   link: 'https://github.com/RConsortium/r-advisory-database/blob/main/'
   editable: False
 
-- name: 'redhat-stage'
+- name: 'redhat'
   versions_from_repo: False
-  type: 1
-  bucket: 'osv-test-cve-osv-conversion'
-  directory_path: 'redhat-temporary'
+  rest_api_url: 'https://security.access.redhat.com/data/osv/all.json'
+  type: 2
+  directory_path: 'data/osv'
+  detect_cherrypicks: False
```

Comment:
I don't know what this setting does.

Reply:
It's only relevant for [analysing Git repositories](https://github.com/google/osv.dev/blob/fe6155f7cfa0e5df0ae1ef20c7b16f5c20bebed1/osv/impact.py#L120), and I don't believe the records being published have any Git commit ranges on them, and this is generally the correct approach as far as I understand cherry picking and the analysis of them...